packages/editor-ui/src/components/ExpressionEditorModal/ExpressionEditorModalInput.vue

```diff
@@ -44,6 +44,7 @@ export default mixins(expressionManager, completionManager, workflowHelpers).ext
 			autocompletion(),
 			Prec.highest(
 				keymap.of([
+					{ key: 'Tab', run: acceptCompletion },
```

Comment:
does this make sense in expression editor too? would it block users from inserting tabs if they are making a few paragraphs

Reply:
Users already cannot tab to indent in the expression editor, tab moves focus to expression editor modal output. `acceptCompletion` will only run when completions are open, else CM will keep cycling through to the rest of the keymap.